### PR TITLE
Change example configuration protocol on loopback IP (RIPD-886)

### PR DESCRIPTION
**Pull request by miguelportilla**
Disable SSL on loopback addresses specified in the example configuration file.

@reviers @vinniefalco @nbougalis

### CODE REVIEW

**Comment by vinniefalco** [comment]
We need to know what to do with the websocket ports

**Comment by miguelportilla** [comment]
I confirmed the problem exists on secured websocket connection to a local rippled instance. The problem is easily addressed in the same manner. The port_wss_admin protocol was changed from wss to ws.

**Comment by MarkusTeufelberger** [comment]
Please also rename the name to "port_ws_admin" and include a line or two to explain that this does NOT use SSL/TLS and that it is not safe or in any way recommended to expose the port to the internet that way. Without TLS remote users don't have any guarantee that there is no MITM or that data has not been messed with on the way to them.

File: doc/rippled-example.cfg [hunk 1]
```diff
@@ -842,7 +842,7 @@ port_wss_admin
 port = 5005
 ip = 127.0.0.1
 admin = 127.0.0.1
-protocol = https
+protocol = http
```
Comment (ximinez) [inline doc/rippled-example.cfg]: Doesn't uniport allow us to specify both secured and unsecured protocols, and decides by examining the message?

I think the rippled client code only looks at the first option here, but would

File: doc/rippled-example.cfg [hunk 2]
```diff
@@ -853,7 +853,7 @@ protocol = peer
 port = 6006
 ip = 127.0.0.1
 admin = 127.0.0.1
-protocol = wss
+protocol = ws
```
Comment (ximinez) [inline doc/rippled-example.cfg]: Same here. Could we do 
`protocol = ws, wss`
?

File: doc/rippled-example.cfg (hunk 1 above)
Comment (miguelportilla) [inline doc/rippled-example.cfg]: Yes it does and I had tried that configuration but unfortunately it seems to choose https.

**Comment by miguelportilla** [comment]
Good catch with the name @MarkusTeufelberger. I don't think a comment is necessary as that example is using the loopback adapter at 127.0.0.1 which avoids security issues. The importance and use of secured universal ports could probably use some commentary in that section.

**Comment by MarkusTeufelberger** [comment]
You also need to change the name in the declaration (line 836), it would also be nice if you could remove the "v" on line 152 while you're at it.

Also I'd like to suggest to rename these 2 connection sections to [port_rpc_admin_local] and [port_ws_admin_local] - this would indicate that they are only to be used in a local setting and might cut down on user errors.

File: doc/rippled-example.cfg (hunk 1 above)
Comment (ximinez) [inline doc/rippled-example.cfg]: Oh well. At least you tried it.

File: doc/rippled-example.cfg [hunk 3]
```diff
@@ -842,18 +842,18 @@ port_wss_admin
...
-protocol = https
+protocol = http
 
 [port_peer]
 port = 51235
 ip = 0.0.0.0
 protocol = peer
 
-[port_wss_admin]
+[port_ws_admin]
```
Comment (nbougalis) [inline doc/rippled-example.cfg]: You need the update the stanza in the `[server]` section too.

Comment (miguelportilla) [inline doc/rippled-example.cfg]: Which line(s)?

**Comment by miguelportilla** [comment]
@MarkusTeufelberger Thanks! done.

**Comment by nbougalis** [comment]
:+1:

**Pull request closed without merging**